release/dist/cli/cli.go

```diff
@@ -190,12 +192,25 @@ func parseSigningKey(path string) (crypto.Signer, error) {
 }
 
 var genKeyArgs struct {
+	root     bool
```

Comment:
This leaves open the (unlikely) possibility that both might be true, or both false. Arguable, but what do you think of:

```
type KeyType int

const (
  KeyTypeInvalid KeyType = iota
  KeyTypeRoot
  KeyTypeSigning
)

var genKeyArgs struct {
  keyType KeyType
```

Maybe I'm just having `enum` feelings today.

Reply:
I was considering doing the enum in the distsign package, but decided to just use different func/type names as a stronger compile-time check.
If we want to keep the flags as `--root/--signing` and not `--type=root|signing`, I don't think an enum here would do much. We'd basically convert the flags to enum, and then convert the enum to function call right after. The intermediate steps seems unnecessary :thinking: